static/src/javascripts/projects/common/modules/navigation/search.js

```diff
@@ -48,9 +56,10 @@ define([
         }
 
         this.focusSearchField = function () {
-            var $input = $('input.gsc-input');
+            var $input = $('input.gsc-input'),
+                inputLenght = $input.length;
```

Comment:
Also why are you putting this into a `var`? You only seem to use it in one place.